definitions/checks/sca_only.rb

```diff
@@ -0,0 +1,42 @@
+class Checks::CheckOrganizationContentAccessMode < ForemanMaintain::Check
+  metadata do
+    label :check_organization_content_access_mode
+    description 'Check if any organizations are using EntitlementContentAccess mode before upgrading'
+    tags :pre_upgrade
+
+    confine do
+      feature(:katello)
+    end
+  end
+
+  def run
+    with_spinner('Checking organization content access modes') do
+      output = execute_rake_task
+      orgs_to_migrate = parse_orgs_to_migrate(output)
+
+      assert(
+        orgs_to_migrate.empty?,
+        "The following organizations are using EntitlementContentAccess mode:\n"\
+        "#{orgs_to_migrate.join(', ')}\n"\
+        "As of Katello 4.12, EntitlementContentAccess mode is removed, and these organizations"\
+        " will be automatically migrated to SimpleContentAccess mode during the upgrade."\
```

Comment:
I think we use this or the non-capitalized version ("simple content access"), in most user-facing places I've seen. cc @bangelic

```suggestion
        " will be automatically migrated to Simple Content Access during the upgrade."\
```

Reply:
I made the change, but I'll leave this review item open for now in case @bangelic wants to give input